ui/lib/utils/openLink.ts

```diff
@@ -0,0 +1,18 @@
+import { NavigateFunction } from 'react-router'
+import React from 'react'
+
+export default function openLink(
+  url: string,
+  ev: React.MouseEvent<HTMLElement>,
+  navigate: NavigateFunction
+) {
+  if (ev.metaKey || ev.altKey || ev.ctrlKey) {
+    // open in a new tab
+    window.open(`/#${url}`, '_blank')
```

Comment:
This does not work in PD integration mode (the prefix is /dashboard)

Reply:
Got it, let me think about it.